templates/validator/tables.html

```diff
@@ -276,8 +276,8 @@
 {{ define "validatorDepositsTable" }}
   {{ with .Data }}
     <div class="table-eth1">
-      <h4 class="my-3">Eth1 Deposits</h4>
-      <h6 class="">This table displays the deposits made to the Eth1 deposit contract.</h6>
+      <h4 class="my-3">Ethereum Deposits</h4>
```

Comment:
```suggestion
      <h4 class="my-3">Ethereum Staking Deposits</h4>
```
May be valuable to add some clarity to this title.